app/Views/errors/cli/error_exception.php

```diff
@@ -41,20 +41,11 @@
             $function .= $padClass . $error['function'];
         }
 
-        $args = implode(', ', array_map(static function ($value) {
-            switch (true) {
-                case is_object($value):
-                    return 'Object(' . get_class($value) . ')';
-
-                case is_array($value):
-                    return count($value) ? '[...]' : '[]';
-
-                case $value === null:
-                    return 'null'; // return the lowercased version
```

Comment:
Do we need this comment?

Reply:
I think not, if needed, rector can't cover it automatically as moved inside match Expr

Reply:
No problem. 
My question is "Is it worth to keep the comment manually?".

Reply:
I think not

Reply:
Agreed, unnecessary.